README.md

```diff
@@ -1 +1,29 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+How to set up a microverse project
+
+> Description the project.
+> This is a simple html and css project to improve my knowledge of creating a pull request for a code review on my Microverse project.
+
+## Built With
+
+- Html, Css
+
+
```

Comment:
- [ ] Descriptive and professional README file! 👍🏼 It would be great if you can add a `Getting started` section that includes information about how to get a local copy of your project up and running to make it even better! This would guide other people who want to collaborate on your project to have an easy setup and a good starting guide. 💯 🤓 

Kindly add this section to your README file! 🗒️ 
(e.g. clone the repo, install project dependencies, open index.html with a browser, etc.)

![image](https://user-images.githubusercontent.com/77069011/197851165-e45b2ba5-3a14-48ea-844b-831c90404a30.png)